### ISSUE TITLE
Custom CSS is being rendered on Settings Page in CP

### ISSUE THREAD

**Issue by RitterKnightCreative** [wbrowar/craft-admin-bar#7]
Great plugin.

One small issue: I usually put a rule like this so admin bar won't obscure the website content:

``html { margin-top: 40px !important; }``

This works well, except on Craft's own settings page, the custom CSS from adminbar is actually being spit out:

![screen shot 2018-04-09 at 5 56 41 pm](https://user-images.githubusercontent.com/3707869/38527264-725b07ea-3c1f-11e8-9ea7-924205d6eba5.png)

Not sure if that's a Craft bug (or an adminBar one) as the CSS doesn't seem to be rendering anywhere else in the CP except on the main settings page.

**Comment by wbrowar** [comment]
Hi @RitterKnightCreative! I'll look into why the custom code is appearing in the CP, but in the mean time I think I have a solution for you. If you change your CSS to the following, this should prevent the margin-top to affect the CP:

`html.adminbar-on { margin-top: 40px !important; }`

You could also do something like this to make sure that the height is correct on desktop and mobile (if you're using it on mobile):

```css
html.adminbar-on { margin-top: var(--adminbar-height-mobile) !important; }
@media (min-width: 601px) {
  html.adminbar-on { margin-top: var(--adminbar-height-desktop) !important; }
}
```

Would you mind letting me know if that works? I'll work on the CSS bug for the next release.

**Comment by RitterKnightCreative** [comment]
Will, that actually worked well! Thank you.

**Comment by RitterKnightCreative** [comment]
While on the subject, I found I needed to add:

`` .adminbar__overlay { line-height:45px; }``

to the CSS or the items in the top don't line up properly. 

![screen shot 2018-04-13 at 3 58 59 pm](https://user-images.githubusercontent.com/3707869/38757750-c8129916-3f33-11e8-82ea-f9713dd9c79b.png)

Also, it seems adminbar is overriding my font stack. (I was using Google Fonts IIRC on this particular site but I think TypeKit is affected as well, will check to see.) When I log out, it goes back to normal.
